Formula/dzr.rb

```diff
@@ -0,0 +1,22 @@
+class Dzr < Formula
+  desc "Command-line Deezer.com player"
+  homepage "https://github.com/yne/dzr"
+  url "https://github.com/yne/dzr/archive/refs/tags/230504.tar.gz"
+  sha256 "b7c50f1e75dd9cc75b56ede33ab4e9e1d09cbfce09bfaa2e9715f99896361447"
+  license "Unlicense"
+  head "https://github.com/yne/dzr.git", branch: "master"
+
+  depends_on "dialog"
+  depends_on "jq"
+  depends_on "mpv"
+  depends_on "openssl@1.1" # to match curl one
```

Comment:
On macOS, `curl` will not use OpenSSL, so should this be used only on Linux?

Reply:
dzr really need openssl to decrypt audio track

When I tried latest openssl version I got a dependency mismatch so I rolled back to 1.1